### PR TITLE
JBPM-6836: Move View Process Instances from details toolbar to proces…

### PR DESCRIPTION
**Pull request by nmirasch**
…s definitions actions column

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac could you take a look? Thanks!

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java [hunk 1]
```diff
@@ -128,17 +128,10 @@ public void initColumns(ExtendedPagedTable extendedPagedTable) {
...
-                                                       "");
-        final TextHeader header = new TextHeader(constants.Actions());
-        header.setHeaderStyleNames("text-center");
-        columnMeta.setHeader(header);
-        columnMetas.add(columnMeta);
+        columnMetas.add(actionsColumnMeta);
 
+        extendedPagedTable.addSelectionIgnoreColumn(actionsColumnMeta.getColumn());
         extendedPagedTable.addColumns(columnMetas);
-        extendedPagedTable.setColumnWidth(actionsColumn,
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java]: I believe we still need to set the column width for actions, to match the other list tables.

**Comment by cristianonicolai** [comment]
@nmirasch overall looks good, just a minor comment regarding the actions column width.

**Comment by cristianonicolai** [comment]
@nmirasch compilation failed, it probably needs to rebase.

**Comment by nmirasch** [comment]
@cristianonicolai rebased!

**Comment by cristianonicolai** [comment]
Compilation fix: https://github.com/kiegroup/jbpm-wb/pull/1005

**Comment by cristianonicolai** [comment]
@nmirasch one more rebase :P

**Comment by nmirasch** [comment]
@cristianonicolai rebased again! :P

**Pull request merged**